server/game/GameActions/GainHonorAction.ts

```diff
@@ -23,20 +23,41 @@ export class GainHonorAction extends PlayerAction {
 
     canAffect(player: Player, context: AbilityContext, additionalProperties = {}): boolean {
         let properties: GainHonorProperties = this.getProperties(context, additionalProperties);
-        return properties.amount === 0 ? false : super.canAffect(player, context);
+        var wouldGainAnyHonor = properties.amount !== 0;
+
+        if (!wouldGainAnyHonor)
+            return false;
+
+        const honorGainLimitPerPhase = player.getEffects(EffectNames.LimitHonorGainPerPhase).reduce((total, value) => total + value, 0);
```

Comment:
Shouldn't this be a min, not a reduce?  If you have 3 effects, one which limits honor gains to 2, one to 4, and one to 5, you want to limit gains to 2, not 11.